JS/Lesson15/sinan/script.js

```diff
@@ -116,14 +129,49 @@ function checkIfStringIsUpperCase(stringToCheck) {
 	}
 }
 
-checkIfStringIsUpperCase('Write a program that takes in');
-checkIfStringIsUpperCase('write a program that takes in');
+checkIfStringIsUpperCase('Exercise 4.1:','Write a program that takes in');
+checkIfStringIsUpperCase('Exercise 4.2:','write a program that takes in');
 
 // 5. Write a program that takes in a number and checks if it's positive, negative, or zero using if else statement.
+function checkNumberPositive(checkNumber) {
+	if (checkNumber > 0){
+		return 'checkNumber is positive';
+	}
+	else if (checkNumber < 0){
+		return 'checkNumber is negative';
+	}
+	else if (checkNumber === 0){ 
+		return 'checkNumber is equal to zero';
+	}
+	else {
+		return 'NaN';
+	}
+}
+console.log('Exercise 5:',checkNumberPositive(-55));
+
 
 // 6. Write a program that takes in a year and checks if it's a leap year (is divisible by 4 and not divisible by 100 OR divisible by 400) or not using if else statement.
+function LeapYear(checkYear) {
+	if (checkYear %4 === 0){
```

Comment:
The check should be a bit more complex than this: check if a number is divisible by 4 **and not divisible by 100** OR divisible by 400